docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc

```diff
@@ -2511,6 +2511,8 @@ subquery is empty, the predicate is false.
 | <<in_predicate,IN Predicate>> | Determines if a sequence of values is equal to any of the sequences of
 values in a list of sequences.
 | <<like_predicate,LIKE Predicate>> | Searches for character strings that match a pattern.
+| <<regexp_predicate,REGEXP Predicate>> | Searches for character strings that match a pattern, The pattern 

```

Comment:
Suggested wordsmith: "Searches for character strings that match an extended regular expression."